modules/wechat_qrcode/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 set(the_description "WeChat QR code Detector")
-ocv_define_module(wechat_qrcode opencv_core opencv_imgproc opencv_dnn WRAP java objc python js)
+ocv_define_module(wechat_qrcode opencv_core opencv_imgproc opencv_objdetect opencv_dnn WRAP java objc python js)
```

Comment:
Why do you need `opencv_objdetect` dependency for the patch?

Reply:
`opencv_objdetect` need to generate synthetic QR code test